userdocs/diagnostics/diagnostics.md

```diff
@@ -4,4 +4,84 @@
    @TechnologyRoot
 }
 
-Documentation on diagnostics emitted by the compiler.
\ No newline at end of file
+Documentation on diagnostics emitted by the compiler and settings that control them.
+
+
+## Diagnostics
+
+Swift diagnostics are classified into errors and warnings. Warnings can only be silenced within
+source in an intentional manner, eg. adding `_ =` for an unused function result.
```

Comment:
Not strictly true: some warnings are enabled using command-line options and can only be silenced by removing (other) options. E.g. `StrictLanguageFeatures`.

```suggestion
Swift diagnostics are classified into errors and warnings. Warnings can only be silenced in an intentional manner, eg. adding `_ =` for an unused function result.
```